nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -505,9 +723,203 @@ def _impl(inputs, attr, params):
     'DepthwiseConv2dNative'             : _depthwise_conv(),
     'Shape'                             : _shape(),
     'Sigmoid'                           : AttrCvt('sigmoid'),
+    'Fill'                              : _fill(),
+    'GatherV2'                          : _gather_v2(),
+    'StridedSlice'                      : _stridedSlice(),
+}
+
+# _convert_map_rnn defines maps of rnn operator name to
+# converter functor(callable) for 1 to 1 mapping.
+_convert_map_rnn = {
+    'LSTMBlockCell'                     : _LSTMBlockCell(),
 }
 
 
+class RecurrentNetworks(object):
+    """Recurrent network layer handlers.
+
+    Unlike normal operators, recurrent network operators have layer concept.
+    Same operators will be called multiple times (based on number of layers)
```

Comment:
Does layer refer to steps? or multiple cells (layers) in stacked rnn? The doc here is not very clear.

Reply:
@Huyuwei here, layers refers to multiple cells in RNN stack. I have rephrased the comments